### ISSUE TITLE
Feature start broke when invoked with the base argument

### ISSUE THREAD

**Issue by nvie** [nvie/gitflow#86]
Hmm. Somehow feature start broke recently, when invoked with the optional `base` argument:

```
$ feature start foo HEAD
Switched to a new branch 'feature/HEAD'

Summary of actions:
- A new branch 'feature/HEAD' was created, based on 'HEAD'
- You are now on branch 'feature/HEAD'

Now, start committing on your feature. When done, use:

     git flow feature finish HEAD
```

There's no branch `feature/foo`, but `feature/HEAD`. This must be argument parsing related. Have to look into this when I have time.

**Comment by pilotmoon** [comment]
Same issue with hotfix start. It parses the last arg into VERSION and second arg into BASE. So with two args, VERSION and BASE are being set to the second arg and the first is ignored.

**Comment by nvie** [comment]
Fixed in 68e53aae80f9b3b74edffd1ecb5458063bb9d6ed.
